CMakeLists.txt

```diff
@@ -221,6 +221,13 @@ option(ENABLE_SIRIUS_TEST "Enable test for fingerid (SiriusAdapter)" ON)
 #------------------------------------------------------------------------------
 option(ENABLE_FINGERID_TEST "Enable test for fingerid (SiriusAdapter)" OFF)
 
+#------------------------------------------------------------------------------
```

Comment:
better place for this would be `cmake/compiler_flags.cmake`

Reply:
oh, I didn't mean to include this change here. I will remove that.